src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalysisScope.cs

```diff
@@ -39,11 +39,6 @@ internal class AnalysisScope
 
         public bool ConcurrentAnalysis { get; }
 
-        /// <summary>
-        /// True if we need to categorize diagnostics into local and non-local diagnostics and track the analyzer reporting each diagnostic.
-        /// </summary>
-        public bool CategorizeDiagnostics { get; }
```

Comment:
This flag became unused after recent code refactoring to remove partial analysis state tracking.